configure.ac

```diff
@@ -716,7 +716,7 @@ AC_ARG_ENABLE([realms],
 AC_ARG_ENABLE([rtadv],
   AS_HELP_STRING([--disable-rtadv], [disable IPV6 router advertisement feature]))
 AC_ARG_ENABLE([irdp],
-  AS_HELP_STRING([--disable-irdp], [disable IRDP server support in zebra (enabled by default if supported)]))
+  AS_HELP_STRING([--enable-irdp], [enable IRDP server support in zebra]))
```

Comment:
This commit `zebra, build: disable irdp by default` shouldn't be part of this PR.

Reply:
It's a separate commit, I don't think it matters too much in which PR it is merged. In fact, I think it's better to keep it in this one, because this NB conversion was the trigger for the decision to disable it.

Reply:
I'm not sure about which auto-highlight tool you're talking, but I suppose they all work based on commits and their subject lines, not on PRs/merge commits. Yes, this PR works without this commit, because IRDP is a separate module that must be explicitly loaded (`-M irdp`) even if it's built.

Reply:
I was talking about a release note generator, I would think that commits would be too fine a granularity. @Jafaral would know about this if it exists.